contracts/SpokePool.sol

```diff
@@ -62,7 +89,7 @@ abstract contract SpokePool is Testable, Lockable, MultiCaller {
     ) Testable(timerAddress) {
```

Comment:
in the constructor params might it be better to put the timer last? we always do this in all the other UMA contracts. wdyt?

Reply:
sure